game/scripts/npc/abilities/doom_bringer_infernal_blade.txt

```diff
@@ -29,7 +29,7 @@
       "01"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "burn_damage"                                     "25 30 35 40 60 90"
+        "burn_damage"                                     "25"
```

Comment:
I think we should scale this in oaa

Reply:
not necessary since the % hp damage scales up quite a bit and they also removed scaling of the flat damage